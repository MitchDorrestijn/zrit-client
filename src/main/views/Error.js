/**
 * React related imports
 */
import React from 'react';

/**
 * Self made components imports
 */
import TextContent from '../components/TextContent';

/**
 * This component renders the error page
 */
const ErrorPage = (props) => {
	return (
    <div>
      <TextContent title="Oh nee!">
				Als u deze pagina ziet betekent het dat er iets niet helemaal goed gegaan is. Ga terug en probeer het opnieuw!
      </TextContent>
    </div>
	);
}

export default ErrorPage;
