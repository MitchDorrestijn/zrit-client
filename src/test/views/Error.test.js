import React from 'react';
import {shallow} from 'enzyme';
import toJSON from 'enzyme-to-json';
import Error from '../../main/views/Error';

const wrapper = shallow(<Error />);

test('<Error /> should match snapshot', () => {
  expect(toJSON(wrapper)).toMatchSnapshot();
});

test('<Error /> should render one <TextContent> component', () => {
  expect(wrapper.find('TextContent').length).toBe(1);
});
