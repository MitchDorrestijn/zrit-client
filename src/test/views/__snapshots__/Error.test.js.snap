// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error /> should match snapshot 1`] = `
<div>
  <TextContent
    title="Oh nee!"
  >
    Als u deze pagina ziet betekent het dat er iets niet helemaal goed gegaan is. Ga terug en probeer het opnieuw!
  </TextContent>
</div>
`;
